enclave_wrangler/dataset_download.py

```diff
@@ -46,9 +43,13 @@ def getTransaction(datasetRid: str, ref: str = 'master') -> str:
 
     response = requests.get(url, headers=HEADERS,)
     response_json = response.json()
-    print(response_json)
+
+    if response.status_code >= 400:
+        raise RuntimeError(json.dumps(response_json))
```

Comment:
Added this. I was getting `KeyError` because it expected a `200`, but I got an error (`400`).

@Sigfried Here's where I'm having the most trouble, and I'd like you to take a look at this and let me know if you know what's going on.

[concept_set_version_items_rv_edited_mapped page](https://unite.nih.gov/workspace/data-integration/dataset/details/ri.foundry.main.dataset.e7941080-8df0-4392-96c5-82fc2f84e2a7/master):
<img width="1533" alt="Screen Shot 2022-05-11 at 1 17 17 PM" src="https://user-images.githubusercontent.com/13045020/167911448-6b621018-11f3-4e79-bf72-da4d665c2917.png">

error:
```
{'errorCode': 'INVALID_ARGUMENT', 'errorName': 'Catalog:BranchesNotFound', 'errorInstanceId': '537796d5-230c-481c-8a45-2e56f309c3d9', 'parameters': {'datasetRids': '[ri.foundry.main.dataset.e7941080-8df0-4392-96c5-82fc2f84e2a7]', 'branchIds': '[master]'}}
```

It seems to be saying that the `master` branch is not recognized, but from what I can see, there is a `master` branch.